docs/guides/use-team-admin-notifications.md

```diff
@@ -0,0 +1,176 @@
+# Use team admins notifications
+
+Team admins notifications help to set up email notifications for different event changes. It uses following preferences:
+
+<attributes title="Preferences">
+
+<attribute name="attending" type="boolean" default="false">
+
+Send notification if user is attending the event you have created.
+
+</attribute>
+<attribute name="declined" type="boolean" default="false">
+
+Send notification if user declined the event you have created.
+
+</attribute>
+<attribute name="event_cancelled" type="boolean" default="false">
+
+Send notification if the event cancelled by others.
+
+</attribute>
+<attribute name="event_cancelled_admin" type="boolean" default="false">
+
+Send notification if the event is cancelled by you.
+
+</attribute>
+
+<attribute name="event_confirmed" type="boolean" default="false">
+
+Send notification if the event is confirmed by other.
+
+</attribute>
+
+<attribute name="event_confirmed_admin" type="boolean" default="false">
+
+Send notification if the event is confirmed by you.
+
+</attribute>
+
+<attribute name="new_event" type="boolean" default="false">
+
+Send notification about new events created by others (but not on your Vyte page).
+
+</attribute>
+
+<attribute name="new_event_admin" type="boolean" default="false">
+
+Send notification about new events created by you.
+
+</attribute>
+
+<attribute name="new_message" type="boolean" default="false">
+
+Send notification about new messages sent by others in the events you participate in or have created.
+
+</attribute>
+
+<attribute name="new_suggestion" type="boolean" default="false">
+
+Send notification about new suggestions made by others in the events you participate in or have created.
+
+</attribute>
+
+<attribute name="new_suggestion_own" type="boolean" default="false">
+
+Send notification about new suggestions made by you in the events you participate in or have created.
+
+</attribute>
+
+<attribute name="vote_update" type="boolean" default="false">
+
+Send notification about updates of other's votes.
+
+</attribute>
+
+<attribute name="vote_confirmation" type="boolean" default="false">
+
+Send notification about your own votes.
+
+</attribute>
+
+<attribute name="vyteme_created_admin" type="boolean" default="false">
+
+Send notification, when new event is created from your Vyte page. As an owner of Vyte page you are always considered as an event creator.
+
+</attribute>
+
+## Set up your team admin preferences
+To set up your team admin preferences you can use the `/teams/:team_id/admin-preferences` endpoint with your team `_id` and POST method. If some preferences are not defined they will be set up `false` par default.
+
+```shell screen-hidden
+curl --request POST 'https://api.vyte.in/teams/:team_id/admin-preferences' \
```

Comment:
always show /v2/ endpoints in the documentation

Reply:
done 5cbfb535239a5ff71cbe94684857a838d764f48b